contents/blog/posthog-vs-hotjar.md

```diff
@@ -0,0 +1,301 @@
+---
+date: 2023-05-08
+title: "In-depth: PostHog vs Hotjar"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-vs-hotjar/posthog-vs-hotjar.jpg
+featuredImageType: full
+author:
+    - lior-neu-ner
+category: General
+tags:
+    - Comparisons
+---
+
+Want to understand the difference between Hotjar and PostHog? Here's the short answer:
+
+1. Hotjar enables you to see how customers are using your product with session replays and heatmaps. It also provides widgets for collecting feedback and tools for scheduling customer interviews. Hotjar is built for marketers and user experience (UX) researchers.
+
+2. PostHog is an all-in-one product improvement platform for product analytics, session recording, feature flags, in-app prompts, and more. PostHog is built for engineers and product teams.
+
+Now it's time for the long answer...
+
+In this guide, we'll cover:
+
+- Hotjar and PostHog [feature comparison](#feature-comparison)
+- [Session replay](#session-replay) capabilities
+- [Privacy](#privacy-compliance) and [security](#admin--security)
+- [Frequently asked questions](#frequently-asked-questions)
+
+## How is PostHog different?
+
+### 1. PostHog is an all-in one platform
+
+Hotjar focuses mainly on session replays, heatmaps, and user research. That means you need to adopt additional tools for things like product analytics, feature management, and A/B testing. PostHog integrates all these features into one platform (in addition to session replays and heatmaps). PostHog is like mission control for your product.
+
+### 2. PostHog is built for engineers
+
+ As such, PostHog includes many powerful features that aren’t available in tools like Hotjar, which is built for more general audiences.
+
+We built PostHog to support technically-savvy product managers and engineers — especially [engineers with a product focus in their role](/blog/what-is-a-product-engineer). Although anyone can use PostHog ([just ask our marketing team](blog/posthog-marketing), our goal is to help engineers be better at product by giving them the insights and tools they need to ship impactful features at speed.
+
+### 3. PostHog is open source
+
+Because we build for engineers first, we're also open source. You can check out [PostHog’s source code](https://github.com/PostHog), [build integrations](/docs/apps/build) or [other services](/blog/how-we-built-an-app-server) on top of the product, and even [give feedback or interact with the team via GitHub](https://github.com/PostHog). Transparency, including transparent pricing, is one of our core values.
```

Comment:
Best to switch the feedback link out to https://app.posthog.com/home#supportModal as that's the preferred feedback form now

Reply:
This link is only available for users that have already created a PostHog app